cartridge/failover/raft.lua

```diff
@@ -58,7 +58,21 @@ local function get_appointments(topology_cfg)
         ::next_rs::
     end
 
-    appointments[vars.replicaset_uuid] = vars.leader_uuid
+    -- If there is still no leader in the replicaset, appoint the first
+    -- electable instance as a leader.
+    for replicaset_uuid, _ in pairs(replicasets) do
+        if appointments[replicaset_uuid] == nil then
```

Comment:
The failover.disable doesn't change the membership payload.

If we disabled instance, raft_term and leader_uuid will remain the same.

If there were instances and Raft in the replicaset, the instances will keep the same raft_term and leader_uuid, and the old leader will remain as it was with Raft.

Therefore, it's better not to check for nil.